mora/service/orgunit.py

```diff
@@ -424,119 +506,157 @@ def edit_org_unit(unitid):
 
-    # Always update gyldighed
-    update_fields.append((
-        mapping.ORG_UNIT_GYLDIGHED_FIELD,
-        {'gyldighed': "Aktiv"}
-    ))
 
-    if keys.NAME in data.keys():
-        attrs = mapping.ORG_UNIT_EGENSKABER_FIELD.get(original)[-1].copy()
-        attrs['enhedsnavn'] = data[keys.NAME]
+@blueprint.route('/ou/<uuid:unitid>/create', methods=['POST'])
+def create_org_unit_relation(unitid):
+    """Creates new unit relations
 
-        update_fields.append((
-            mapping.ORG_UNIT_EGENSKABER_FIELD,
-            attrs,
-        ))
+    .. :quickref: Unit; Create relation
 
-    if keys.ORG_UNIT_TYPE in data.keys():
-        update_fields.append((
-            mapping.ORG_UNIT_TYPE_FIELD,
-            {'uuid': data[keys.ORG_UNIT_TYPE]['uuid']}
-        ))
+    :statuscode 200: Creation succeeded.
 
-    if keys.PARENT in data.keys():
-        update_fields.append((
-            mapping.PARENT_FIELD,
-            {'uuid': data[keys.PARENT]['uuid']}
-        ))
+    :param employee_uuid: The UUID of the employee.
 
-    payload = common.update_payload(new_from, new_to, update_fields, original,
-                                    payload)
+    All requests contain validity objects on the following form:
 
-    bounds_fields = list(
-        mapping.ORG_UNIT_FIELDS.difference({x[0] for x in update_fields}))
-    payload = common.ensure_bounds(new_from, new_to, bounds_fields, original,
-                                   payload)
+    :<jsonarr string from: The from date, in ISO 8601.
+    :<jsonarr string to: The to date, in ISO 8601.
 
-    c.organisationenhed.update(payload, unitid)
+    .. sourcecode:: json
 
-    return flask.jsonify(unitid)
+      {
+        "from": "2016-01-01T00:00:00+00:00",
+        "to": "2018-01-01T00:00:00+00:00",
+      }
+
+    Request payload contains a list of creation objects, each differentiated
+    by the attribute 'type'. Each of these object types are detailed below:
+
+
+    **Address**:
+
+    :<jsonarr string type: ``"address"``
+    :<jsonarr object address_type: The type of the address, exactly as
+        returned by returned by
+        :http:get:`/service/o/(uuid:orgid)/f/(facet)/`.
+    :<jsonarr string address: The value of the address field. Please
+        note that as a special case, this should be a UUID for *DAR*
+        addresses.
+
+    .. sourcecode:: json
+
+      [
+        {
+          "address": "1234567890",
+          "address_type": {
+            "example": "5712345000014",
+            "name": "EAN",
+            "scope": "EAN",
+            "user_key": "EAN",
+            "uuid": "e34d4426-9845-4c72-b31e-709be85d6fa2"
+          },
+          "type": "address",
+          "validity": {
+            "from": "2016-01-01T00:00:00+00:00",
+            "to": "2018-01-01T00:00:00+00:00"
+          }
+        }
+      ]
+
+    """
+
+    reqs = flask.request.get_json()
+
+    if not isinstance(reqs, list):
+        return flask.jsonify('Root object must be a list!'), 400
```

Comment:
Why not do the same thing here, of wrapping the payload in a list, if it isn't?

Reply:
YAGNI — wrapping is for compatibility, and this function is new?

Reply:
I was merely thinking in terms of keeping our API consistent.
If the other instance of wrapping in a list is strictly an undocumented compatiblity thing, then let's just keep this as it is.